### PR TITLE
PK Machine gun

### PR DESCRIPTION
**Pull request by NanakoAC**
This PR is branched off of the Autofire PR, so that must be merged first, then there will be conflicts to resolve here
Adds the PK machine gun to the serbian armoury, replacing the old L6 SAW

Its functionally just a reskin of the saw, identical stats and ammo, but it is much nicer looking, thanks to wonderful sprites by Shazbot

video of it in action
https://drive.google.com/file/d/1LQvCXvDLa-NNZuasTxgo6YkR0t1j4du2/view?usp=drivesdk

Also returns some of the old crate sprites to the dmi with the old_prefix, just to keep them in the file. I picked the green weapon crate specifically for merc gear and it got replaced in the adding of the new crates

### CODE REVIEW

**Comment by Wesreidau** [comment]
The PK machine gun fires 7.62x54R ammunition which is significantly more powerful and entirely incompatible with 7.62x39 ammunition found in the AK-47 platform. The PK is firing a full powered rifle cartridge, while the AK-47 fires an intermediate cartridge. There is a difference in armor or structural penetration which could be modelled here. In its current state, PK ammunition boxes can be used to carry ammunition to refill AK magazines.

File: code/game/objects/items/weapons/storage/deferred.dm [hunk 1]
```diff
@@ -142,21 +143,21 @@
 
 /obj/item/weapon/storage/deferred/crate/saw
 	name = "infantry support crate"
-	desc = "a crate containing two L6 SAW light machine guns, and 640 rounds of 7.62mm ammunition"
-	icon_state = "weaponcrate"
-	initial_contents = list(/obj/item/weapon/gun/projectile/automatic/l6_saw = 2, /obj/item/ammo_magazine/ammobox/a762 = 8)
+	desc = "a crate containing two Pulemyot Kalashnikova light machine guns, and 640 rounds of 7.62mm ammunition"
```
Comment (Wesreidau) [inline code/game/objects/items/weapons/storage/deferred.dm]: desc = "a crate containing two Pulemyot Kalashnikova light machine guns, and 800 rounds of 7.62mm ammunition"

Comment (NanakoAC) [reply code/game/objects/items/weapons/storage/deferred.dm]: Our AK isn't actually an AK47, Its an AKMysteriousNumber onto which we can handwave whatever properties we desire.

In practical terms, this thing has 600 RPM and thusly an insane damage output, it doesn't also need more powerful ammo

Comment (Wesreidau) [reply code/game/objects/items/weapons/storage/deferred.dm]: Or the AK-47 can have a weaker ammunition with the PK's output remaining the same. However you like it.

**Pull request merged**